model/src/main/java/com/basistech/rosette/dm/internal/TextWrapper.java

```diff
@@ -40,6 +40,7 @@
  * on token indexes.  The underlying <code>Text</code> object contains only
  * character offsets.
  */
+@SuppressWarnings("deprecation")
```

Comment:
```
[~/views/annotated-data-model-internal (master)]
$ find . -name "TextWrapper*"
./tools/src/test/java/com/basistech/rosette/dm/internal/TextWrapperTest.java
```

`TextWrapper` lives in public github annotated-data-model, but `TextWrapperTest`
lives in the private basis annotated-data-model-internal repo.  I think we should
move the test to the public repo.  (This is still in an "internal" package until we
decide how much we like it.  In particular, the iteration through mentions in order
is pretty compelling now.)